### ISSUE TITLE
Content review for notification preferences, MVP notification items and groups

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#23907]
We received the initial list of notifications for our MVP from VA Profile and VA Notify.  Can you please review the content in the group name and communication items columns in the tab titled `content/IA - for review`, and we'll discuss in 5/6 content office hours?  Please reach out with any questions. Thanks!

[communications-permissions-configuration-data-worksheet-042821-edits.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/6390413/communications-permissions-configuration-data-worksheet-042821-edits.xlsx)

**Comment by andaleliz** [comment]
Our team is due to have these finalized for VA Profile by 5/13. I need to have the VA Notify team review them as well and I want to give them a couple of days. Is it possible to get feedback from you by Monday AM, so I have time to update the mock-ups?

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA Heads up that I've added one more thing to this for your review - shouldn't be intensive, but need your feedback.

**Comment by mnorthuis** [comment]
Per our conversation this morning, here are some thoughts on the groupings:
- I would keep the groupings broader to begin with.  Given we are unsure what types of notifications may come down the line and in what order, and how much users will interact with this, broader groups will allow us to more easily bucket things initially, and can be broken down into more granular options as we see the need arise. 
- I would recommend using language that is consistent with how we label things in the benefits experience - i.e. instead of "financial", possibly use "payments and debt", since that is how we refer to those items on the site. 

I think the groups you have after our discussion this morning align with the above recommendations pretty well. I would look to Danielle on any feedback regarding the names/labels:
- Application, claims and decision reviews: This would include any notifications regarding in progress applications and forms, status updates on submitted forms, and updates on decision review and appeal requests. 
- Your health care:  This bucket would be anything related to their ongoing health care services and information such as appointments, prescriptions, secure messaging, etc. 
- Payments and debt: This would include any compensation payments received, as well as any debt owed, including medical copays. 
- General information and updates from VA:  any other VA general notifications or announcements.

**Comment by DanielleThierryUSDSVA** [comment]
Yes, these buckets align with our discussion and content strategy/IA recommendations. Thank you, @mnorthuis!

**Comment by DanielleThierryUSDSVA** [comment]
Good morning, @andaleliz! 

1. Yes, agree. Let's leave the note out for now and test.

2. Agree. Slight tweaks:

This will delete your mobile phone number across many VA records. You may no longer get some or all VA notifications by text message. You can always come back to your profile later if you want to add your mobile number back in.

3. As we talked about, I would recommend broader buckets for the notifications to avoid confusion and overlap. I think we're good to go with testing these bucket labels:

- Applications, claims, and decision reviews
- Your health care
- Payments and debts
- General VA updates and information

**Comment by andaleliz** [comment]
Thanks @DanielleThierryUSDSVA! 

Confirming you're good with the notification names I've proposed as well? They're in the spreadsheet, but for ease of review:

- Reminder to finish an application in progress (group: Applications, claims, and decision reviews)
- Appeals hearing reminder (group: Applications, claims, and decision reviews)
- COVID-19 updates (group: General VA updates and information)
- Compensation and pension payment scheduled (group: Payments and debts)
- Appointment reminders (group: Your health care)
- Prescription shipment and tracking information (group: Your health care)

**Comment by andaleliz** [comment]
Thank you so much! `Applications, claims, decision reviews, and appeals` feels kind of long to me, but it seems necessary, and I get that it's temporary.

I've sent these out to VA Notify for review, and will update the rest of the mock-ups accordingly.
